pkg/scheduler/scheduler.go

```diff
@@ -75,7 +72,7 @@ func (s *Service) startPod(suite v1alpha1.ClusterTestSuite, def v1alpha1.TestDef
 	p.Labels = def.Spec.Template.Labels
 	p.Annotations = def.Spec.Template.Annotations
 
-	p.GenerateName = TestingPodGeneratedName
+	p.GenerateName = fmt.Sprintf("%s-%s-", suite.Name, def.Name)
```

Comment:
can you explain to me why this approach is better than the previous one? We can filter pods by labels (v1alpha1.LabelKeySuiteName, v1alpha1.LabelKeyTestDefName). How do you want to ensure that the generated name does not exceed limits?

Reply:
This helps when you have multiple suites run at once. Thanks to that you don't have to think about the labels, `kubectl get pods` prints you nice list of all run tests. You instantly see which of them have failed.

For example: 
```
kubectl get pods
NAME                                       READY     STATUS      RESTARTS   AGE
testsuite-all-api-controller-mdhtm         0/1       Completed   0          2d
testsuite-all-kubeless-e2e-dxkm2           0/1       Error       0          2d
testsuite-all-kubeless-integration-h79jm   0/1       Completed   0          2d
```

It doesn't provide any functional enhancement. It's purely readability improvement.

Reply:
One of these days I would see tests within a suite run in a dedicated namespace. The prefix would be redundant them. Now, however, I like it because when you run multiple test suites at once pods are nicely grouped together.